frontend/packages/knative-plugin/src/topology/listView/SinkUriListViewNode.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import { observer } from '@patternfly/react-topology';
+import {
+  OdcBaseNode,
+  TopologyListViewNode,
+  TypedResourceBadgeCell,
+} from '@console/dev-console/src/components/topology';
+import { DataListCell } from '@patternfly/react-core';
+
+interface KnativeServiceListViewNodeProps {
+  item: OdcBaseNode;
+  selectedIds: string[];
+  onSelect: (ids: string[]) => void;
+}
+
+const ObservedSinkUriListViewNode: React.FC<KnativeServiceListViewNodeProps> = ({
+  item,
+  selectedIds,
+  onSelect,
+  children,
+}) => {
+  const sinkUri = item.getResource()?.spec?.sinkUri;
+
+  const labelCell = (
+    <DataListCell className="odc-topology-list-view__label-cell" key="label" id={sinkUri}>
+      {<TypedResourceBadgeCell key="type-icon" kind={item.getResourceKind()} />}
+      {sinkUri}
```

Comment:
If it does not find this, we are back to where we are with this bug 🤔 

Not sure what we could put here though.